starter/source/elements/initia/initia.F

```diff
@@ -821,7 +820,7 @@ SUBROUTINE INITIA(IPARG      ,ELBUF         ,MS               ,IN             ,V
       IF(BCS%NUM_WALL > 0)THEN
         CALL INIT_BCS_WALL(IGRNOD,NGRNOD,NUMNOD,ALE_CONNECTIVITY,MULTI_FVM,
      .                     IXS,NIXS,NUMELS, IXQ,NIXQ,NUMELQ, IXTG,NIXTG,NUMELTG, N2D,
-     .                     NGROUP,NPARG,IPARG,IWORKING_ARRAY,IPRI)
+     .                     NGROUP,NPARG,IPARG,IPRI,ITAB)
```

Comment:
@Schwoertzig 

Can you double check? seems ITAB argument is not present in INIT_BCS_WALL definition.

Thanks,
Sébastien